### PR TITLE
[Quarry] Add basic crawler

### PR DESCRIPTION
**Pull request by kabiroberai**
To invoke the crawler, use `python3 manage.py scrape`. Note that although the crawler does indeed retrieve the webpage's contents, I chose not to log them because it would lead to unnecessary spam.

### CODE REVIEW

**Comment by palash-taneja** [comment]
Hey @kabiroberai, can you clean up the code. It seems like you put the code for the landing page and the crawler in the same PR

File: search_engine/src/management/commands/scrape.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand, CommandError
+from .scraper.crawler import crawl
+
+
+class Command(BaseCommand):
+    help = 'Scrape the web'
+
+    def handle(self, *args, **options):
```
Comment (MatejMecka) [inline search_engine/src/management/commands/scrape.py:8]: Missing Docstring :)

Comment (kabiroberai) [reply search_engine/src/management/commands/scrape.py:8]: PEP-8 only mandates docstrings for public-facing functions and classes, so this is a non-issue.

**Comment by kabiroberai** [comment]
@PalashTanejaPro the landing page code was rebased from the `landing-page` branch (ref #93). Some of the changes introduced in that branch are also used here, which is why the rebase was necessary. The crawler code itself was added in a37683d.

**Review by palash-taneja — APPROVED** [review]
LGTM great code 👍

**Pull request closed without merging**